src/vivarium_cluster_tools/vipin/perf_report.py

```diff
@@ -189,6 +191,86 @@ def print_stat_report(perf_df: pd.DataFrame, scenario_cols: list):
     logger.info(f"\n{report_df}")
 
 
+def append_perf_data_to_central_logs(perf_df: pd.DataFrame, output_directory: Path) -> None:
+    """Append performance data to the central logs."""
+    perf_df = perf_df.reset_index()
```

Comment:
Do you need to make a copy of `perf_df` or is your intention to also append this stuff to the non-central logs?

Reply:
I should make a copy. I was misunderstanding scope/namespaces and thought this wouldn't modify the dataframe outside of the function, but it was sloppy to use the same name even if that were the case. Will update!